### PR TITLE
add database table and indentation

### PR DESCRIPTION
**Pull request by Lisonallie**
Here I created the database with the name Notes. I wanted to name it something simple and central to the project itself that I could easily remember and write.
----
I created an ID as a sort of identifier of who wrote the post and it has the primary key because I don't want the ID number to ever duplicate.
----
The names for each section are representations of how logical I could try to be by saying what action  each field name will be intended to log. I capitalized the first part of each name because it makes it look more like a name to me. This way I won't mistake it for something else in later versions.
The authors Note_authors varchar length is 70 because in a previous exercise I was told for first name and last name the standard was 35 chars. Seeing as I have the first name and last name together, I made the length max 70 chars.
----
Note_titles varchar is limited to 255 because of the previous lesson as well. I learned that 255 was the basic ending standard for small bouts of text. I like the number as well because it's long enough that it doesn't feel short and doesn't feel too long at the same time. A paragraph should not be a title. The default value is also NOT NULL because I don't want the user to have the option to fill in a blank title.
----
Text_entries varchar is limited to 2048 because the standard, as I read from the previous exercise, is  1024. I found this maybe to be too little for notes knowing from my own experiences with planning trips and laying out presentations and such in my own notes app. So I decided to double that number and indeed limit it because very long notes also take up a lot of space.
----
Date_posted I have as 'date' because I want to show only the year/month/day. I find that specifying the post to the minute and second is a little too much information in this case so I wanted to keep it simple.
----

### CODE REVIEW

**Review by RafSwiggers — APPROVED** [review]
Very structured and readable. Great basic set-up.

**Review by FloorJulesSegers — COMMENTED** [review]
Very clean and understandable Alisson!

File: database.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE IF NOT EXISTS Notes;
+
+CREATE TABLE IF NOT EXISTS Notepad
+(
+    ID              int             NOT NULL        PRIMARY KEY AUTO_INCREMENT,
+    Note_authors    varchar (70)    DEFAULT NULL,
+    Note_titles     varchar (255)   DEFAULT NOT NULL,
```
Comment (ghost) [inline database.sql:7]: 255 is maybe lil to big for a title

**Review by tinevancorenland — COMMENTED** [review]
I like the names: clear descriptions. 
I like that the titles have to be filled in but text entries not: sometimes a note can be just that. 

Why are authors and dates optional? 
Does it add value that you add the **defaults**?
Does it add value that you add the **null**?

**Comment by Lisonallie** [comment]
> I like the names: clear descriptions.
> I like that the titles have to be filled in but text entries not: sometimes a note can be just that.
> 
> Why are authors and dates optional?
> Does it add value that you add the **defaults**?
> Does it add value that you add the **null**?

Authors and dates are optional because they're not necessary for the actual note-posting process. Really only the title and note are necessary for that. I wanted to add a bit extra to challenge myself.
I had a problem in the previous project because I didn't have a default value for my columns so I just add a default now to have less problems.
Null is a non-value. It's not 0 and not white space so it's perfect for something that has no value until it's filled in.

File: database.sql (hunk 1 above)
Comment (Lisonallie) [reply database.sql:7]: For me this is only the maximum length that is allowed. If someone -really- wants to make a title with 255 they can. But 256 will be too long.

**Pull request merged**